sbt/src/main/scala/stryker4s/log/SbtLogger.scala

```diff
@@ -1,25 +0,0 @@
-package stryker4s.log
-
-import sbt.{Level as SbtLevel, Logger as SbtInternalLogger}
-
-class SbtLogger(sbtLogger: SbtInternalLogger) extends Logger {
-
-  def log(level: Level, msg: => String): Unit = sbtLogger.log(toSbtLevel(level), msg)
```

Comment:
I don't see this class moved anywhere. How does logging now work inside the plugins?

Reply:
SbtLogger is moved to module stryker4jvm-plugin-sbt. The SbtLogger class is private but a new FansiSbtLogger class has been created.